### PR TITLE
Added title with link to MR

### PR DESCRIPTION
**Pull request by alexandrugheorghe**
Made the link to the merge request more accessible.

### CODE REVIEW

**Comment by alexandrugheorghe** [comment]
First PR! 🎉 🥇

File: cmd/merge.go [hunk 1]
```diff
@@ -250,6 +250,8 @@ func (c *client) getMergeRequestCommits(mr *gitlab.MergeRequest) (commits []*git
 
 func (c *client) notifySlackChannel(channel string, a *mergeArgs, mr *gitlab.MergeRequest, em *EmailMappings) (err error) {
 	attachment := slack.Attachment{
+		Title: fmt.Sprintf("Click for MR"),
+		TitleLink: fmt.Sprintf("%s", mr.WebURL),
 		Pretext: fmt.Sprintf("🚀 merging `%s` to `%s` in *%s*", a.sourceRef, a.targetRef, a.project),
 		Footer:  fmt.Sprintf("%s/diffs", mr.WebURL),
```
Comment (mvisonneau) [inline cmd/merge.go]: should we remove the footer ? 🤔 @alexandrugheorghe

**Comment by mvisonneau** [comment]
@alexandrugheorghe, the lint job is also failing unfortunately!

File: cmd/merge.go (hunk 1 above)
Comment (alexandrugheorghe) [reply cmd/merge.go]: > @alexandrugheorghe, the lint job is also failing unfortunately!

How do I run the linter locally?

Comment (mvisonneau) [reply cmd/merge.go]: `make lint`

Comment (mvisonneau) [reply cmd/merge.go]: and `make fmt` to fix the issue :)

**Pull request merged**